main.py

```
@@ -0,0 +1,250 @@
+    host = f"{name}.{namespace}.svc"
+    url = f"ws://{host}:{port}"
+    try:
+        async with connect(url):
+            log = "Successfully connected"
+    except Exception as e:
+        log = str(e)
+
+    if "401" in log or log == "Successfully connected":
+        status = "UP"
+    else:
+        status = "FAIL"
```

Ruff (0.11.9)

Use ternary operator `status = "UP" if "401" in log or log == "Successfully connected" else "FAIL"` instead of `if`-`else`-block

Replace `if`-`else`-block with `status = "UP" if "401" in log or log == "Successfully connected" else "FAIL"`

(SIM108)